### DISCUSSION TITLE
Can I use the placement module to change the widget attached on the Content Zone ?

### DISCUSSION THREAD

**Discussion by KshitizGIT** [OrchardCMS/OrchardCore#9425] [Q&A]
In the BlogTheme, I added a Paragraph Widget to the "Content" Zone in the "Always" Layer. But the paragraph is displayed ahead of the actual content i.e either content of Blog or BlogPost...
Can I use the placements module to render the paragraph after the contents of Blog or BlogPost ? If yes, how do I do this ?

**Comment by KshitizGIT** [comment]
Well, it seems the purpose of the placement is to customize position logic for the Fields and ContentPart in any ContentItem. It would have been helpful if the docs mentioned it out right..
What I was trying to do was to use the placements module to customize position logic of a Widget in the Content Zone. Is there any way to customize position logic of ContentItem, Widgets in a zone ?

**Reply by jtkech** [reply]
You need to add a new zone (e.g. `AfterContent`) through the admin UI and render it at the right place in your theme layout.
